src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -107,6 +107,17 @@ private bool EmitNupkg(PublishRoot root)
                 root.Operations.Delete(TargetPath);
             }
 
+            // If this is a wrapper project, we need to generate a lock file before building it
+            if (IsWrappingAssembly())
+            {
+                var success = Restore(root, publishProject: this, restoreDirectory: project.ProjectDirectory)
```

Comment:
The output is already suppressed in the `Restore()` helper method:
https://github.com/aspnet/dnx/pull/2512/files#diff-e073db6edc86fb957608d9a10b4e521dR436

All NuGet package references of this project should be installed when you restore for the main project (i.e. the project referencing this project). So the answer is "no".

BTW, the `Restore()` helper method has
```c#
restoreCommand.IgnoreMissingDependencies = true; 
```
so missing package dependencies shouldn't be problems.